crates/stc_ts_type_checker/tests/conformance/statements/for-inStatements/for-inStatementsInvalid.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 14,
-    matched_error: 3,
-    extra_error: 1,
+    required_error: 6,
+    matched_error: 11,
+    extra_error: 2,
```

Comment:
This is caused by the multiple `x`.

Reply:
<img width="642" alt="image" src="https://user-images.githubusercontent.com/29931815/218386542-dc01aff1-821f-4bdf-a63f-5811fe39a174.png">
<img width="642" alt="image" src="https://user-images.githubusercontent.com/29931815/218386589-21dbbba2-381c-4a7d-87cf-64a930a60369.png">

Reply:
I think this is crazy... I'll merge it.